doc/CreateFlow.md

```diff
@@ -43,3 +51,57 @@ In order to build the Node-RED flow for the CASIS satellite tracker, you will us
 * Details are available in the [Create Worldmap UI video](https://ibm.box.com/s/glu64qw4wfic9im642vcapgwwzn2dcvg)
 
 ![video-to-gif](source/video/createWorldmapUI.gif)
+
+### Create the Watson Assistant Chat UI
+
+* Grab the following nodes in order, and wire them up in a row. The following 3 groups represent 3 rows:
+
+```
```

Comment:
maybe have this as ascii art, but in it's current state i think it's confusing more than anything.